jwst/resample/resample.py

```diff
@@ -296,15 +306,22 @@ def resample_many_to_many(self):
             if not self.in_memory:
                 # Write out model to disk, then return filename
                 output_name = output_model.meta.filename
+                if self.output_dir is not None:
+                    output_name = os.path.join(self.output_dir, output_name)
                 output_model.save(output_name)
+                if self.mk_output_list:
+                    self.output_list.append(output_name)
```

Comment:
It looks like `output_list` contains the same items as `output_models`. What required adding the new list?

Reply:
Isn't that the same thing that's in `output_models`? line 315 contains:
```
self.output_models.append(output_name)
```
which appends the same variable as:
```
self.output_list.append(output_name)
```

Reply:
yes, I agree that the 2 lines are appending the name, however when you print self.output_models you get a model container and not a list of strings

Reply:
Thanks! I did miss that point, and it looks like there is an issue with resample (one that exists on main) where it creates the container with an unknown argument with presumably the expectation that he models are not used:
https://github.com/spacetelescope/jwst/blob/9e857a52f8cd2dd58a04e6b71751fa15fca32556/jwst/resample/resample.py#L186
As resample appends filenames it's possible to either:
- disable `return_open` (which I expect is what the `open_models` was intended to do) and indexing the container will return the filenames
- access `_models` (which will contain the filenames)
I went with the second approach in https://github.com/penaguerrero/jwst/pull/1